sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/Messages.java

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventhubs;
+
+import com.azure.core.util.logging.ClientLogger;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * I18n messages loaded from the messages.properties file located within the same package.
+ */
+public enum Messages {
+    ;
+    private static final ClientLogger LOGGER = new ClientLogger(Messages.class);
```

Comment:
@JonathanGiles Did we have a checkstyle for not having static final loggers? I see why it's used here though in that .getProperties() method below. 🤔

Reply:
I think we did have a check for it....not sure why it isn't triggering.

Reply:
i excluded the check for messages to report errors during static initialisation.  

<!-- Requires static access to logger to report errors while loading i18n messages (from within a static initializer )-->
  <suppress checks="com.azure.tools.checkstyle.checks.GoodLoggingCheck" files="Messages.java"/>